fixtures/pages-workerjs-directory/tests/index.test.ts

```diff
@@ -23,8 +23,8 @@ describe.concurrent("Pages _worker.js/ directory", () => {
 			fetch(`http://${ip}:${port}/static`).then((resp) => resp.text())
 		).resolves.toContain("static");
 		await expect(
-			fetch(`http://${ip}:${port}/other-script`).then((resp) => resp.text())
-		).resolves.toContain("test");
+			fetch(`http://${ip}:${port}/other-script.js`).then((resp) => resp.text())
+		).resolves.toContain("other-script-test");
```

Comment:
LOL this test used to pass even before this change because the error stack trace contained the word "test".